### PR TITLE
docs: Add CONTRIBUTING guidelines for broken CI jobs

### PR DESCRIPTION
**Pull request by pedroerp**
Following up on offline discussion, adding instructions to CONTRIBUTING.md file
to instruct contributors not to ignore broken CI jobs, and create github Issues
when appropriate.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,8 +85,16 @@ The contribution process is outlined below:
...
-   * Create/submit a Github PR and tag the reviewers identified in Step 3.
+   * Make sure the PR passes **all CI tests**.
+     * Do not ignore red CI signals, even if they seem preexisting.
+     * If you believe a red CI signal is unrelated to your change, please search for
+     existing Issues reporting this particular test. They should contain the test name
+     in the Issue title.
+     * If an Issue already exist, add a comment containing the link to your failed CI job.
+     * If an Issue does not exist, please create one with the title "Broken CI \<test\_name\>",
+     tagging the appropriate maintainers for that component.
+   * Once all CI signals are green, create/submit a Github PR and tag the reviewers
```
Comment (mbasmanova) [inline CONTRIBUTING.md:96]: > Once all CI signals are green, create/submit a Github PR

I don't think it is possible to launch CI without creating a PR first.

**Review by mbasmanova — APPROVED** [review]
Thanks.

File: CONTRIBUTING.md (hunk 1 above)
Comment (majetideepak) [reply CONTRIBUTING.md:96]: Create/submit a GitHub PR and once all CI signals are green, tag the reviewers identified in Step 3.

**Review by majetideepak — APPROVED** [review]
Thanks, Pedro.

File: CONTRIBUTING.md (hunk 1 above)
Comment (pedroerp) [reply CONTRIBUTING.md:96]: Yep, that is what I meant. Let me rephrase.
